workbench/header.php

```diff
@@ -50,6 +50,11 @@
               "Workbench is currently undergoing maintenance. The service may be intermittently unavailable during this time.</div><br/>";
 }
 
+if (WorkbenchConfig::get()->isConfigured("displayInstallWarningMessage")) {
```

Comment:
I will check to see if this setting is on in the production app.

Reply:
@ntotten - I don't see `displayInstallWarningMessage` in the code base (or in https://github.com/forceworkbench/forceworkbench/blob/master/workbench/config/defaults.php).

Where did you get this config from?

Reply:
It must be working somehow since it shows up for you but I'm just curious where that config is.

Reply:
PHP! It's all dynamic. I didn't add it yet to production.

Reply:
It's not showing up in the review app as well - https://workbench-ps-staging-pr-823.herokuapp.com/login.php